### DISCUSSION TITLE
Port Movements overhaul commit?

### DISCUSSION THREAD

**Discussion by 6AKU66** [Papela/Nitrox-Cracked-Mod#28] [General]
https://github.com/SubnauticaNitrox/Nitrox/pull/2189
https://www.youtube.com/watch?v=S02Edz2Exe4
Developers improved sync with vehicles. Do you plan to update your fork?

**Comment by Papela** [comment]
I've seen that there have been new commits in the official repository, so when I have free time I'll try to get a new update out. I hope to get it out by mid or late February.
